### ISSUE TITLE
[med] JAWS+Chrome: Email verification: Confirmation message 'link has been resent' not announced

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5465]
## Action Performed:
1. Using JAWS+Chrome, open https://staging.new.expensify.com/r/78379550.
2. Press Tab key to focus the 'Enter your phone or email' field and provide a valid email address.
3. Press Tab key to focus the 'Continue' button and press Enter key.
4. Press Tab key to focus the link 'Resend link' and press Enter key to activate it.
5. Note what is displayed and announced.

## Expected Result:
The displayed confirmation message 'Link has been resent' should be also announced.

## Actual Result:
The displayed confirmation message 'Link has been resent' is not announced by screen reader.

HTML:
`<div dir="auto" class="css-901oao" style="color: rgb(3, 212, 124); font-family: GTAmericaExp-Regular; font-size: 13px; line-height: 18px; margin-bottom: 4px;">Link has been re-sent</div>`

## Workaround:
No workaround

## Area issue was found in:
Email verification page

## Failed WCAG checkpoints
4.1.3

## User impact:
Screen reader users will not know whether the link have been sent or not when status is not announced.

## Suggested resolution:
Use the aria-live attribute to announce the displayed confirmation message 'Link has been resent' once it appears on the screen. See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134568292-7ff376b2-d883-4a7f-b6b3-3a3593e2b9ba.mp4

![Bug5180765_Status_message](https://user-images.githubusercontent.com/88733897/134568305-d72a60e0-73f1-44fa-84bd-d87cfcd575eb.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
